litmus/director/TCID-DIR-OP-INSTALL-OPENEBS-LIMIT-RESOURCE/test.yml

```diff
@@ -27,13 +27,61 @@
           shell: cat /etc/secret-volume/password
           register: password
 
-        ## Checking if openebs is already installed
+        ## Checking if openebs is already installed in the cluster
         - name: Check if openebs is already installed
           shell: kubectl get deploy --all-namespaces -l name=maya-apiserver --no-headers | awk '{print $1}' | wc -l
           args:
             executable: /bin/bash
-          register: maya_api
-          failed_when: "{{ maya_api.stdout }} != 0"
+          register: count_maya_api
```

Comment:
here we are not checking if the component is present

Reply:
Here we are checking for maya-api server if it is present the further we are fetching the ns of that component and passing that ns to the cleanup script